.github/workflows/rust.yml

```diff
@@ -67,7 +67,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-latest, ubuntu-24.04-arm, macos-latest]
```

Comment:
While there might be some use for running clippy for aarch64 linux because of the c_char thing, running tests feels like an unnecessary use of resources.